ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -156,7 +158,9 @@ def get_alias(prefix: str, test=False, set_alias_suffix: str | None = None) -> X
     if set_alias_suffix:
         return XmlAttribute(f"{prefix}:{set_alias_suffix}")
     if test:
-        return XmlAttribute(f"{prefix}:{datetime.datetime.now(tz=pytz.utc)}")
+        entropy = "".join(secrets.choice(string.ascii_letters + string.digits) for _ in range(4))
```

Comment:
I think might be cleaner
```
''.join(secrets.choices(string.ascii_letters + string.digits, k=4))
```

Reply:
that would be nice but it errors?

```
>>> import string, secrets
>>> ''.join(secrets.choices(string.ascii_letters + string.digits, k=4))
Traceback (most recent call last):
  File "<python-input-3>", line 1, in <module>
    ''.join(secrets.choices(string.ascii_letters + string.digits, k=4))
            ^^^^^^^^^^^^^^^
```

<img width="670" height="338" alt="image" src="https://github.com/user-attachments/assets/6417f184-fc6f-49ec-8e6b-b60007038a46" />